Move some content around

katrinundfritz opened the pull request.

katrinundfritz commented on src/stories/foundations/design_principles.stories.mdx:
```diff
@@ -4,7 +4,19 @@ import { Meta } from '@storybook/addon-docs'
+
+## Vision
+
+Our Vision is to create seamless, beautiful and professional design. Accessible for everyone. We want our users to find quickly what they need, have fun while using it and engage highly through them with others to create impact for good.
```
@inesczuia I don't think the vision is really helpful here for different reasons:

> Our Vision is to create seamless, beautiful and professional design.
Unclear what seamless, beatiful and professional actually should mean. Sounds a bit random (like something everyone wants) as well.

> Accessible for everyone.
Repeats what is already listed in the principles.

>  We want our users to find quickly what they need, have fun while using it
This is just parts of the definition of UX in other less precise words. 

> engage highly through them
Unclear what that means.

katrinundfritz commented on src/stories/foundations/design_principles.stories.mdx:
```diff
@@ -4,7 +4,19 @@ import { Meta } from '@storybook/addon-docs'
+
+## Mission:
+
+We want clarity, system and consistency in design. We want that once set design rules are used as strong guidelines and have leverage in discussions. Our design should be a facilitator for better communication and processes and not an obstacle in the way of innovation. The user / human should always be at the center of every decision.
```
@inesczuia The last sentence only repeats what was already listed in the principles.

```suggestion
We want clarity, system and consistency in design. We want that once set design rules are used as strong guidelines and have leverage in discussions. Our design should be a facilitator for better communication and processes and not an obstacle in the way of innovation.
```

katrinundfritz commented:
@inesczuia I started reordering some stuff and so moved some design content around as well.

My intention was to reduce the _About the design system_ page really only to why we have it and how it can be used (how it is structured). When moving the design content I was not sure if we really need all of it.

inesczuia commented:
Hey  @Katrin Fritz ***@***.***> most of these sections
were still WIP and Katharina and I wanted to go into specifics also
together with brand.
I'll have a look at what you did and might re-reorder it then :D
*Ines Czuia*
UX/UI Designerin
***@***.***
www.betterplace.org
<http://www.google.com/url?q=http%3A%2F%2Fwww.betterplace.org%2F&sa=D&sntz=1&usg=AFQjCNHfyQeY7uQVbuw8A0d6irrQE548pA>

*gut.org <http://gut.org> gemeinnützige Aktiengesellschaft*, Schlesische
Str. 26, 10997 Berlin
Vorstand: Timo Kladny, Björn Lampe, Felix Oldenburg (Sprecher)
Vorsitzender des Aufsichtsrats: Till Behnke
Registergericht: Amtsgericht Berlin Charlottenburg | HRB 126785 B |
Geschäftssitz: Berlin


On Wed, Mar 15, 2023 at 5:56 PM Katrin Fritz ***@***.***>
wrote:

> @inesczuia <https://github.com/inesczuia> I started reordering some stuff
> and so moved some design content around as well.
>
> My intention was to reduce the *About the design system* page really only
> to why we have it and how it can be used (how it is structured). When
> moving the design content I was not sure if we really need all of it.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/betterplace/betterplace-design-system/pull/51#issuecomment-1470404718>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANJTOGCHLXXKAQ4ULQNSEO3W4HYFXANCNFSM6AAAAAAV4DEBXY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

katrinundfritz commented:
For now I removed the foundations part and moved it's content to the UX Design Guide draft. We will find out if we need something more than the guides in the future.

I as well added a components area.
